src/amo/pages/Home/index.js

```diff
@@ -301,38 +311,62 @@ export class HomeBase extends React.Component {
 
         <LandingAddonsCard
           addonInstallSource={INSTALL_SOURCE_FEATURED}
-          addons={shelves.featuredExtensions}
-          className="Home-FeaturedExtensions"
-          header={i18n.gettext('Featured extensions')}
-          footerText={i18n.gettext('See more featured extensions')}
+          addons={shelves.recommendedExtensions}
+          className="Home-RecommendedExtensions"
+          header={
+            enableFeatureRecommendedBadges
+              ? i18n.gettext('Recommended extensions')
+              : i18n.gettext('Featured extensions')
+          }
+          footerText={
+            enableFeatureRecommendedBadges
+              ? i18n.gettext('See more recommended extensions')
+              : i18n.gettext('See more featured extensions')
+          }
           footerLink={{
             pathname: '/search/',
             query: {
               addonType: ADDON_TYPE_EXTENSION,
-              featured: true,
+              featured: enableFeatureRecommendedBadges ? undefined : true,
+              recommended: enableFeatureRecommendedBadges ? true : undefined,
+              sort: enableFeatureRecommendedBadges
+                ? SEARCH_SORT_RANDOM
+                : undefined,
             },
           }}
           loading={loading}
         />
 
         {renderFeaturedCollection(0)}
 
-        {includeFeaturedThemes && (
+        {includeRecommendedThemes && (
           <LandingAddonsCard
             addonInstallSource={INSTALL_SOURCE_FEATURED}
-            addons={shelves.featuredThemes}
-            className="Home-FeaturedThemes"
```

Comment:
This test needs to be updated:

https://github.com/mozilla/addons-frontend/blob/8607c35f1eb8edc469d8bb1b2825ff804512e838/tests/unit/amo/pages/TestHome.js#L183

Reply:
Good catch, thanks. I actually found another one in `test_versions`, which I also fixed.